cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -189,19 +187,23 @@ DDEV_DEBUG=true ddev start -y || ( \
   printf "Start failed.\n" && \
   exit 1 )
 
+host_http_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.services.web.host_http_url' 2>/dev/null)
 http_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.httpURLs[0]' 2>/dev/null)
 https_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.httpsURLs[0]' 2>/dev/null)
 
 header "Curl of site from inside container"
 ddev exec curl --fail -I http://127.0.0.1
 
-header "curl -I of ${http_url} from outside"
+header "curl -I of ${host_http_url} (web container http docker bind port) from outside"
+curl --fail -I ${host_http_url}
```

Comment:
Do we also need to curl for `${host_https_url}` (there is not such variable yet)?

Reply:
The host_https_url is pretty useless, doesn't really demonstrate anything at all, and we don't really want people to use it. I don't think.